app/controllers/index.js

```diff
@@ -12,26 +14,44 @@ export default Controller.extend({
   contributorsSorting: Object.freeze(['balance:desc']),
   contributorsSorted: sort('contributorsWithKredits', 'contributorsSorting'),
 
-  proposals: alias('kredits.proposals'),
-  proposalsOpen: filterBy('proposals', 'isExecuted', false),
-  proposalsClosed: filterBy('proposals', 'isExecuted', true),
-  proposalsSorting: Object.freeze(['id:desc']),
-  proposalsClosedSorted: sort('proposalsClosed', 'proposalsSorting'),
-  proposalsOpenSorted: sort('proposalsOpen', 'proposalsSorting'),
+  contributions: alias('kredits.contributions'),
+  contributionsSorting: Object.freeze(['id:desc']),
+
+  contributionsUnconfirmed: computed('contributions.[]', 'currentBlock', function() {
+    return this.contributions.filter(contribution => {
+      return contribution.confirmedAt < this.currentBlock;
```

Comment:
I think this should be the other way round. 
unconfirmed = confirmedAt > currentBlock

or: 
if the currentBlock is smaller than the block number in confirmedAt then the contribution is not confirmed.

Reply:
```suggestion
      return this.currentBlock < contribution.confirmedAt;
```

Reply:
Aw crap, of course. Brain fart.